### ISSUE TITLE
Alter Tables: ChildApprovals -> CasePeriods

### ISSUE THREAD

**Issue by katelovescode** [pieforproviders/pieforproviders#1739]
## 📋 Summary

AC:
- Rename `ChildApprovals` to `CasePeriods`
- Remove `rate_type` and `rate_id` and all associations to Rates in this model
- `belongs_to` child
- `has_many` attendances

## 🇺🇸 States this change applies to
- [X] ALL
- [ ] Nebraska
- [ ] Illinois

**Comment by nikhilbhatt** [comment]
can I work on this issue?

**Comment by katelovescode** [comment]
Yes!  Please make sure to move it to in progress when you get started.
